runtimes/7.4/Dockerfile

```diff
@@ -5,6 +5,7 @@ LABEL maintainer="Taylor Otwell"
 ARG WWWGROUP
 ARG NODE_VERSION=16
 ARG POSTGRES_VERSION=14
+ARG PHP_VERSION=7.4
```

Comment:
Can you please undo this change? I don't think this belongs in the PR you're attempting to make.